media/wav.ksy

```diff
@@ -0,0 +1,272 @@
+meta:
+  id: wav
+  file-extension: wav
+  title: Microsoft WAVE audio file
+  endian: le
+  license: BSD-3-Clause-Attribution
+  encoding: ASCII
```

Comment:
Actually, this is incredibly messy ;) Microsoft's documentation uses terms like "STR", "BSTR", "ZSTR", etc, which never specify encoding. In practice, most software in Windows uses "ANSI" encoding there, i.e. some arbitrary single-bit encoding, specific to locale of the OS that was used to create the file...